src/js/component.js

```diff
@@ -1573,11 +1594,12 @@ class Component {
     this.clearingTimersOnDispose_ = true;
     this.one('dispose', () => {
       [
+        ['namedRafs_', 'cancelNamedAnimationFrame'],
         ['rafIds_', 'cancelAnimationFrame'],
         ['setTimeoutIds_', 'clearTimeout'],
         ['setIntervalIds_', 'clearInterval']
       ].forEach(([idName, cancelName]) => {
-        this[idName].forEach(this[cancelName], this);
+        this[idName].forEach((val, key) => this[cancelName](key));
```

Comment:
this is to get it to work with both Map and Sets, right?

Reply:
Yes